linera-core/src/node.rs

```diff
@@ -67,6 +61,28 @@ pub trait ValidatorNode {
     async fn subscribe(&mut self, chains: Vec<ChainId>) -> Result<NotificationStream, NodeError>;
 }
 
+/// Turn an address into a validator node (local node or client to a remote node).
```

Comment:
Nit: maybe remove the "local node" reference?

```suggestion
/// Turn an address into a validator node.
```

Reply:
Good catch!